tripal/tripal_galaxy
# Adds 'uid' column to the submission table

Before we changed the API to support non webform submissions, we were able to lookup the user that had submitted the workflow by examining the node that the webform created.  But now that the API supports non webform submissions we need a way to keep track of the user that submitted the workflow.  

This PR adds a 'uid' column to the `tripal_galaxy_workflow_submission` table and adjusts all of the code that previously relied on the node.  

To test this PR, you must
1.  Pull this branch
2.  Run the `drush updatedb`
3.  Check the `tripal_galaxy_workflow_submission` table to make sure it now has a `uid` column, is populated with the proper UIDs and is not null
4.  Sumbit a workflow
     a.  Your workflow should submit, the record in the table should have your UID in it.
     b.  You should receive an email that your submission succeeded or failed. (hopefully succeeded).
5.  After the workflow completes you should check it with the `drush trp-galaxy-status` call and you should get an email on the status if it completed.
6.  Check the Admin > Tripal > Extensions > Galaxy > Usage and make sure the report shows the proper count.
